### PR TITLE
Improve Automated Release Process documentation

### PR DESCRIPTION
**Pull request by mariolenz**
I've done the 9.6.0 and 10.0.0b1 releases today and had some questions about step 5. I think we could improve the documentation here a little bit.

### CODE REVIEW

**Comment by gotmax23** [comment]
I pushed a change to fix markdown syntax for python-markdown. I will leave the rest of my feedback inline. Thanks!

File: docs/automated-process.md [hunk 1]
```diff
@@ -60,13 +60,28 @@ release process. The automated processes uses GitHub Actions to automate the
    branches down to the ansible-core version that is included in the Ansible
    release.
 
-5. Announce the release on the Forum, Matrix, and the mailing list by running
-   the following command in the `${MAJOR_VERSION}` directory of the
-   `ansible-build-data` checkout:
+5. You will need `antsibull` for this step. If you haven't installed it yet,
```
Comment (gotmax23) [inline docs/automated-process.md:63]: I would suggest making installing antsibull locally one step ("Install antsibull locally for the next with pip install ...") and then running the actual command can be separate.

Comment (mariolenz) [reply docs/automated-process.md:63]: Step 5 install antsibull and step 6 doing the announcements? Yes, I could change this.

But I'm AFK for the next couple of days. Feel free to do and merge it yourself. Or wait until I'm back to work on it, probably Sunday or Monday.

Comment (felixfontein) [reply docs/automated-process.md]: I pushed 0261f3c00e7c4eb06aa20d9e1ab88bf2013683cd for that.

Comment (mariolenz) [reply docs/automated-process.md]: @felixfontein LGTM. Thanks!

@gotmax23 What do you think about this?

**Comment by mariolenz** [comment]
@felixfontein @gotmax23 What should we do with this PR? Should we merge, or do you suggest more changes?

**Review by felixfontein — APPROVED** [review]
I think let's merge it. We can still polish it more if necessary.

**Comment by felixfontein** [comment]
@mariolenz thanks for improving the docs!
@gotmax23 thanks for reviewing!

**Pull request merged**